src/test/java/seedu/duke/commands/RestockCommandTest.java

```diff
@@ -38,5 +38,9 @@ public void restockItemLevel() {
         restockParser.run();
         restockParser = new RestockParser("qty/5", inventory);
         restockParser.run();
+        restockParser = new RestockParser("upc/123 qty/5 qty/6", inventory);
+        restockParser.run();
+        restockParser = new RestockParser("upc/123 qty/99999999999999", inventory);
+        restockParser.run();
```

Comment:
Same here, I think you might want to use assertion for Junit tests

Reply:
Same for sell command tests, the test will pass even if something goes wrong so if someone breaks it and just runs all tests (no one reads all the output anyways) they won't know

Reply:
Added assertions for each test case, thanks for point it out everyone!